rss2email/feed.py

```diff
@@ -207,20 +207,25 @@ class Feed (object):
         'digest_post_process',
         ]
 
-    def __init__(self, name=None, url=None, to=None, config=None):
+    def __init__(self, name=None, url=None, to=None, from_email=None, config=None):
         self._set_name(name=name)
         self.reset()
         self.__setstate__(dict(
                 (attr, getattr(self, attr))
                 for attr in self._dynamic_attributes))
         self.load_from_config(config=config)
+        if not from_email is None: self.from_email = from_email
```

Comment:
Conditionally setting or not setting the field is kinda bad style (and might lead to bugs). Instead, I’d read the `config['DEFAULT']['from']` field here if none is set.

Reply:
I now more understand rss2email architecture.
We read default configs in self.load_from_config and after that, we override some values if they're set in parameters. So we have to do it for "from_email" field/parameter.
The coding didn't follow actual style. I'll change it.